criterion-measurement/src/Criterion/Measurement.hs

```diff
@@ -390,6 +404,31 @@ secs k
                | t >= 1e1  = printf "%.2f %s" t u
                | otherwise = printf "%.3f %s" t u
 
+#if defined(__GHCJS__)
+-- atm, cabal haddock --ghcjs fails :(
```

Comment:
Is this due to haskell/cabal#5840? If so, can you leave a link to it in the comments?

Reply:
yes, that's why I didn't bother to haddock those properly (though looks like I did something anyway.)